smithy-waiters/src/test/resources/software/amazon/smithy/waiters/errorfiles/invalid-return-types.errors

```diff
@@ -1,4 +1,4 @@
-[DANGER] smithy.example#A: Waiter `Invalid1`, acceptor 0: Waiter acceptors with a booleanEquals comparator must return a `boolean` type, but this acceptor was statically determined to return a `number` type. | WaitableTraitJmespathProblem
-[DANGER] smithy.example#A: Waiter `Invalid1`, acceptor 1: Waiter acceptors with a stringEquals comparator must return a `string` type, but this acceptor was statically determined to return a `number` type. | WaitableTraitJmespathProblem
-[DANGER] smithy.example#A: Waiter `Invalid1`, acceptor 2: Waiter acceptors with a allStringEquals comparator must return a `array` type, but this acceptor was statically determined to return a `number` type. | WaitableTraitJmespathProblem
-[DANGER] smithy.example#A: Waiter `Invalid1`, acceptor 3: Waiter acceptors with a anyStringEquals comparator must return a `array` type, but this acceptor was statically determined to return a `number` type. | WaitableTraitJmespathProblem
+[DANGER] smithy.example#A: Waiter `Invalid1`, acceptor 0: Waiter acceptors with a booleanEquals comparator must return a `boolean` type, but this acceptor was statically determined to return a `number` type. | WaitableTraitJmespathProblem.Invalid1.0.booleanEquals.number
```

Comment:
Like the other jmespath based validators -- it's too specific to put the path into the event ID. These will be just as fickle as suppressing based on messages since they can and do change.

Reply:
Hmm, I also don't like having the path in the eventId, but I couldn't think of another way to maintain uniqueness. I was under the impression that different JMES paths (for the same waiter and acceptor) could potentially have the exact same issue and therefore have identical eventIds

Reply:
Let's just remove it for now

Reply:
Ok so I can remove any JMES paths from event Ids, but 
1. what you see above is technically the waiter name and acceptor number and comparator name and statically-determined return type
2. Those sort of fields ARE necessary in other WaitableTrait violations in shipped models today. For example the following 3 are raised on a currently shipped model and would otherwise have conflicting eventIds: (sorry for the nonstandard formatting, it was from a script I was running for investigative purposes and goes [target] [eventid] [message] )

com.amazonaws.cloudformation#DescribeStacks WaitableTraitInvalidErrorType Waiter `StackDeleteComplete`, acceptor 1: errorType 'ValidationError' not found on operation. This operation defines the following errors: []

com.amazonaws.cloudformation#DescribeStacks WaitableTraitInvalidErrorType Waiter `StackCreateComplete`, acceptor 6: errorType 'ValidationError' not found on operation. This operation defines the following errors: []

com.amazonaws.cloudformation#DescribeStacks WaitableTraitInvalidErrorType Waiter `StackExists`, acceptor 1: errorType 'ValidationError' not found on operation. This operation defines the following errors: []